tests/test-main.js

```diff
@@ -21,28 +17,7 @@ require.config({
   // we have to kickoff jasmine, as it is asynchronous
   callback: window.__karma__.start,
   paths: {
-    "core/linter-rules/no-headingless-sections":
-      "/base/js/core/linter-rules/no-headingless-sections",
-    "core/linter-rules/no-http-props":
-      "/base/js/core/linter-rules/no-http-props",
-    "core/linter-rules/check-punctuation":
-      "/base/js/core/linter-rules/check-punctuation",
-    "core/linter-rules/local-refs-exist":
-      "/base/js/core/linter-rules/local-refs-exist",
-    "core/l10n": "/base/js/core/l10n",
-    "w3c/linter-rules/privsec-section":
-      "/base/js/w3c/linter-rules/privsec-section",
-    "w3c/l10n": "/base/js/w3c/l10n",
-    "core/LinterRule": "/base/js/core/LinterRule",
-    "core/biblio-db": "/base/js/core/biblio-db",
-    "core/jquery-enhanced": "/base/js/core/jquery-enhanced",
-    "core/pubsubhub": "/base/js/core/pubsubhub",
-    "core/utils": "/base/js/core/utils",
-    "deps/jquery": "/base/js/deps/jquery",
-    "deps/marked": "/base/js/deps/marked",
```

Comment:
Question: Why `deps` aren't needed?

Reply:
Because we don't directly refer `deps` inside tests. Some tests does refer `core/utils` and `w3c/linter` so we need them.